baseplate/experiments/providers/variant_sets/multi_variant_set.py

```diff
@@ -0,0 +1,52 @@
+from .base import VariantSet
+
+
+class MultiVariantSet(VariantSet):
+
+    def __init__(self, variants, num_buckets=1000):
+        self._validate_variants(variants)
+        self.variants = variants
+        self.num_buckets = num_buckets
+
+    def __contains__(self, item):
```

Comment:
🔕 If you check this often, it might be a good idea to just store a `set` of all of the variant names and `return item in self._variant_names`.